packages/orbit-components/src/Tabs/Tabs.stories.tsx

```diff
@@ -65,94 +68,158 @@ export const Compact = () => {
+        </TabList>
+        <TabPanels>
+          <TabPanel active={selected === 0}>Tab 1 content</TabPanel>
+          <TabPanel active={selected === 1}>Tab 2 content</TabPanel>
+          <TabPanel active={selected === 2}>Tab 3 content</TabPanel>
+          <TabPanel active={selected === 3}>Tab 4 content</TabPanel>
+        </TabPanels>
+      </Tabs>
+    );
+  },
 
-export const Playground = ({
-  type,
-  compact,
-  disabled,
-  defaultSelected,
-  firstTabLabel,
-  secondTabLabel,
-  firstTabContent,
-  secondTabContent,
-  fullWidth,
-}) => {
-  return (
-    <Tabs onChange={action("onChange")} defaultSelected={defaultSelected}>
-      <TabList compact={compact} fullWidth={fullWidth}>
-        <Tab disabled={disabled}>{firstTabLabel}</Tab>
-        <Tab type={type}>{secondTabLabel}</Tab>
-      </TabList>
-      <TabPanels>
-        <TabPanel>{firstTabContent}</TabPanel>
-        <TabPanel>{secondTabContent}</TabPanel>
-      </TabPanels>
-    </Tabs>
-  );
+  parameters: {
+    info: "Example of controlled version of Tabs component. Check Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      disable: true,
+    },
+  },
 };
 
-Playground.args = {
-  type: TYPE_OPTIONS.DEFAULT,
-  compact: false,
-  disabled: false,
-  defaultSelected: 0,
-  firstTabLabel: "Tab 1",
-  secondTabLabel: "Tab 2",
-  firstTabContent: "Tab 1",
-  secondTabContent: "Tab 2",
-  fullWidth: false,
-};
+export const Playground: Story = {
+  render: function Render({
+    spacing,
+    defaultSelected,
+    disabled,
+    type,
+    marginTabList,
+    paddingTabList,
+    spacingTabList,
+    marginTabPanel,
+    paddingTabPanel,
+    tabChildrenFirst,
+    tabChildrenSecond,
+    tabPanelChildrenFirst,
+    tabPanelChildrenSecond,
+    ...args
+  }) {
+    return (
+      <Box background="cloudLight">
+        <Tabs defaultSelected={defaultSelected} spacing={spacing}>
+          <TabList
+            spacing={spacingTabList}
+            margin={marginTabList}
+            padding={paddingTabList}
+            {...args}
+          >
+            <Tab type={type} disabled={disabled}>
+              {tabChildrenFirst}
+            </Tab>
+            <Tab type={type} disabled={disabled}>
+              {tabChildrenSecond}
+            </Tab>
+          </TabList>
+          <TabPanels>
+            <TabPanel margin={marginTabPanel} padding={paddingTabPanel}>
+              {tabPanelChildrenFirst}
+            </TabPanel>
+            <TabPanel margin={marginTabPanel} padding={paddingTabPanel}>
+              {tabPanelChildrenSecond}
+            </TabPanel>
+          </TabPanels>
+        </Tabs>
+      </Box>
+    );
+  },
+
+  args: {
+    spacing: SPACINGS.THREE_HUNDRED,
+    defaultSelected: 0,
+    tabChildrenFirst: "Tab 1",
+    tabChildrenSecond: "Tab 2",
+    disabled: false,
+    type: TYPE_OPTIONS.DEFAULT,
+    marginTabList: "10px",
+    paddingTabList: "20px",
+    spacingTabList: SPACINGS.NONE,
+    compact: false,
+    fullWidth: false,
```

Comment:
If I set this to `true`, the tab content is overflowing from the box, can you take a look?

Reply:
It's caused, because there is set a default value for `margin` in Playground story (`20px`). If you change/remove the value, then the indent of tabs is changed.